lib/rubocop/cop/mixin/comments_help.rb

```diff
@@ -62,14 +62,16 @@ def buffer
       # Returns the end line of a node, which might be a comment and not part of the AST
       # End line is considered either the line at which another node starts, or
       # the line at which the parent node ends.
-      # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
+      # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/MethodLength, Metrics/PerceivedComplexity, Lint/DuplicateBranch
       def find_end_line(node)
         if node.if_type? && node.else?
           node.loc.else.line
         elsif node.if_type? && node.ternary?
           node.else_branch.loc.line
         elsif node.if_type? && node.elsif?
           node.each_ancestor(:if).find(&:if?).loc.end.line
+        elsif node.block_type?
```

Comment:
`numblock_type?` is the same, right?

Reply:
I guess so. What is a `numblock_type?` tough?

Reply:
e.g. https://github.com/rubocop/rubocop/pull/11488